python_builtin_functions.py

```diff
@@ -1,12 +1,9 @@
 def myabs(number):
     """ Implementation of built-in `abs` function """
     if number<0:
```

Comment:
Its good. You can also try tertiary operator way as below

`return -number if number<0 else number`